docs/content/en/tasks/monitor/_index.md

```diff
@@ -5,168 +5,318 @@ weight: 6
-```bash
-kubectl rollout status deployment k8ssandra-dc1-stargate
+kubectl create -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/master/bundle.yaml
 ```
 
-**Output:**
+This will install all the CRDs along with a `prometheus` service account (SA), a cluster role and a cluster role binding to the `prometheus` SA.
+It will also create a `prometheus-operator` deployment in the `default` namespace.
 
-```bash
-Waiting for 1 pods to be ready...
-Waiting for 1 pods to be ready...
-Waiting for 1 pods to be ready...
-partitioned roll out complete: 1 new pods have been updated...
-```
+### Deploy the ServiceMonitor resources for the K8ssandraCluster
 
-Then check the overall status of the deployed pods.
+Now that the ServiceMonitor CRD exists in the cluster, we can deploy them for Cassandra, Stargate and Reaper by updating the `.spec.cassandra`, `.spec.stargate` and `.spec.reaper` fields of the K8ssandraCluster CR with:
 
-```bash
-kubectl get pods
+```yaml
+    telemetry:
+      prometheus:
+        enabled: true
 ```
 
-**Output**:
-
-```bash
-NAME                                                READY   STATUS      RESTARTS   AGE
-k8ssandra-cass-operator-766849b497-klgwf            1/1     Running     0          7m33s
-k8ssandra-dc1-default-sts-0                         2/2     Running     0          7m5s
-k8ssandra-dc1-stargate-5c46975f66-pxl84             1/1     Running     0          7m32s
-k8ssandra-grafana-679b4bbd74-wj769                  2/2     Running     0          7m32s
-k8ssandra-kube-prometheus-operator-85695ffb-ft8f8   1/1     Running     0          7m32s
-k8ssandra-reaper-655fc7dfc6-n9svw                   1/1     Running     0          4m52s
-k8ssandra-reaper-operator-79fd5b4655-748rv          1/1     Running     0          7m33s
-k8ssandra-reaper-schema-dxvmm                       0/1     Completed   0          5m3s
-prometheus-k8ssandra-kube-prometheus-prometheus-0   2/2     Running     1          7m27s
-```
+After apply the patch, running `kubectl get servicemonitor -n k8ssandra-operator` should return three ServiceMonitor resources.  
+You can selectively enable service monitor creation for each component without any requirement to enable them all.  
 
-Notice that Grafana is running, as well as other services such as Prometheus.
+*Note: Reaper's telemetry block was added in K8ssandra v1.2.0 and Reaper v3.2.0.*
 
-If you are in a local Kubernetes environment, you can now access the Grafana dashboard with a URL such as:
+### Create/Update a Prometheus deployment
 
-<http://grafana.localhost:8080>
+Create the `prometheus` namespace: `kubectl create namespace prometheus`
+  
+If the Prometheus deployment is to be created in another namespace than the one containing the K8ssandra pods, make sure it has the ability to monitor the pods in the K8ssandra namespace. You may also have multiple CassandraDatacenter resources existing in multiple namespaces, with a single Prometheus instance monitoring all of them.
```

Comment:
These instructions are explicitly creating the Prometheus deployment in the `prometheus` namespace; it seems this instruction should be more imperative than "If the..."?